samples/gym-bipedal-walker-sample/README.md

```diff
@@ -0,0 +1,159 @@
+
+[Curriculum Docs](https://docs.microsoft.com/en-us/bonsai/inkling/keywords/curriculum)
+
+### Curriculum Learning
+
+![](img/multi-lesson.png)
+
+```javascript
+            lesson easy_course {
+                scenario {
+                    stump_height: 0,
+                    obstacle_spacing: 0,
+                }
+                training {
+                    LessonRewardThreshold: 230,
+                }
+            }
+
+            lesson hard_course {
+                scenario {
+                    stump_height: number<0 .. 3>,
+                    obstacle_spacing: number<0 .. 6>,
+                }
+                training {
+                    LessonRewardThreshold: 230,
+                }
+            }
+```
+
+### No Curriculum Learning
+
+![](img/single-lesson.png)
+
+```javascript            
+            lesson hard_course {
+                scenario {
+                    stump_height: number<0 .. 3>,
+                    obstacle_spacing: number<0 .. 6>,
+                }
+            }
+
+```
+
+
+## Array Action
+
+| Action | Continuous Value | Array Elements |
+| ------ | ---------------- | ------- |
+| leg_1_torque[2] | [-1, 1] | [hip joint, knee joint] |
+| leg_2_torque[2] | [-1, 1] | [hip joint, knee joint] |
+
+## States
+
+Num   | Observation              |  Min   |   Max  | Mean
+------|--------------------------|--------|--------|------
+0     | hull_ang                 |  0     |  2*pi  |  0.5
+1     | hull_ang_vel             |  -inf  |  +inf  |  -
+2     | x_velocity               |  -1    |  +1    |  -
+3     | y_velocity               |  -1    |  +1    |  -
+4     | leg_1_hip_angle          |  -inf  |  +inf  |  -
+5     | leg_1_hip_speed          |  -inf  |  +inf  |  -
+6     | leg_1_knee_angle         |  -inf  |  +inf  |  -
+7     | leg_1_knee_speed         |  -inf  |  +inf  |  -
+8     | leg_1_contact            |  0     |  1     |  -
+9     | leg_2_hip_angle          |  -inf  |  +inf  |  -
+10    | leg_2_hip_speed          |  -inf  |  +inf  |  -
+11    | leg_2_knee_angle         |  -inf  |  +inf  |  -
+12    | leg_2_knee_speed         |  -inf  |  +inf  |  -
+13    | leg_2_contact            |  0     |  1     |  -
+14-23 | 10 lidar readings        |  -inf  |  +inf  |  -
+
+## Terminal Conditions
+
+- Episode length greater than 2000
+- Hull collides with ground or obstacle
+
+## Configuration Parameters
+
+| State    | Range                                             |
+| ---------| ------------------------------------------------------- |
```

Comment:
any thoughts on adding the `leg_size` config variables from `env.my_init` here or is that not helpful for this task?

Reply:
The paper only mentions two tracks: stump track and hexagon track for varying difficulty of the sim. The leg_size seems to be glued to the bipdeal walker, short bipedal walker, and short bipedal walker.